cmd/list/machinepool/cmd.go

```diff
@@ -56,9 +57,6 @@ func run(_ *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	if cluster.Hypershift().Enabled() {
-		listNodePools(r, clusterKey, cluster)
-	} else {
-		listMachinePools(r, clusterKey, cluster)
-	}
+	service := machinepool.NewMachinePoolService()
+	service.ListMachinePools(r, clusterKey, cluster, cluster.Hypershift().Enabled())
```

Comment:
Do we need to pass in `cluster.Hypershift().Enabled()` since we pass in the cluster object we can ref that in the func